packages/client/CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove client configuration option `client.network.name`
 - Remove client configuration option `client.debug`
 - Remove (non-functional) client configuration options `autoConnect`, `autoDisconnect`, and `maxRetries`
+- Remove `AuthenticatedAuthConfig` and `UnauthenticatedAuthConfig` interfaces
 
```

Comment:
Add entry somewhere that `auth: {}` is no longer allowed.

Reply:
Fixed